editor/editor_properties_array_dict.cpp

```diff
@@ -882,6 +886,31 @@ void EditorPropertyArray::_reorder_button_up() {
 		array.call("insert", reorder_to_index, value_to_move);
 
 		slots[reorder_to_index % page_length].reorder_button->grab_focus();
+
+		bool last_folding = object->editor_is_section_unfolded(object->get_property_name_for_index(reorder_slot.index));
```

Comment:
The name of the variable seems a bit confusing. It should at least be coherent with the function and use unfold. Maybe `last_unfolding_state`, `previous_unfolded` would be a bit clearer on what it represent.